cmd/jnet/main.go

```diff
@@ -30,17 +32,17 @@ func main() {
 }
 
 func run() error {
-	if err := os.MkdirAll("tmp", 0700); err != nil {
+	if err := os.MkdirAll("tmp", filePerm); err != nil {
 		return errors.Trace(err)
 	}
 
-	dir, err := ioutil.TempDir("", "")
+	dir, err := ioutil.TempDir("", "jnet")
 	if err != nil {
 		return errors.Trace(err)
 	}
 	defer os.RemoveAll(dir)
 
-	if err := ioutil.WriteFile(dir+"/jnet.jsonnet", lib, os.ModePerm); err != nil {
+	if err := ioutil.WriteFile(filepath.Join(dir, "jnet.jsonnet"), lib, filePerm); err != nil {
```

Comment:
Los ficheros son 0600
```suggestion
	if err := ioutil.WriteFile(filepath.Join(dir, "jnet.jsonnet"), lib, 0600); err != nil {
```